osu.Game/Screens/Play/Player.cs

```diff
@@ -354,6 +356,10 @@ protected override void OnEntering(Screen last)
 
             hitRendererContainer.Alpha = 0;
             hitRendererContainer.FadeIn(750, EasingTypes.OutQuint);
+
+            if (!HitRenderer.HasReplayLoaded)
+                using (hudOverlay.ModDisplay.BeginDelayedSequence(2000))
+                    hudOverlay.ModDisplay.FadeOut(200);

```

Comment:
That's one possibility, although I was leaning more towards using hudOverlay.BindHitRenderer.